From 72244e6e6035ffc211ea7a4fce9aa96de8bc1f76 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Caol=C3=A1n=20McNamara?= <caolanm@redhat.com>
Date: Wed, 3 Nov 2010 10:16:06 +0000
Subject: [PATCH] abort doesn't gain us anything here.

abort doesn't gain us anything here except to force abrt automatic bugreports kick
in, and XIOErrors can't be worked around
---
 vcl/unx/source/plugadapt/salplug.cxx |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/vcl/unx/source/plugadapt/salplug.cxx b/vcl/unx/source/plugadapt/salplug.cxx
index 54f9752..ba6bf4a 100644
--- a/vcl/unx/source/plugadapt/salplug.cxx
+++ b/vcl/unx/source/plugadapt/salplug.cxx
@@ -284,7 +284,7 @@ void SalAbort( const XubString& rErrorText )
         std::fprintf( stderr, "Application Error" );
     else
         std::fprintf( stderr, "%s", ByteString( rErrorText, gsl_getSystemTextEncoding() ).GetBuffer() );
-    abort();
+    exit(-1);
 }
 
 const OUString& SalGetDesktopEnvironment()
-- 
1.7.3.1

